src/calculateRentalCost.js

```diff
@@ -28,7 +28,19 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const normalPrice = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  if (days < 3) {
+    return days * 40;
```

Comment:
hmm.. mb 40 = normalPrice
```suggestion
    return days * normalPrice;
```